Tribler/Main/tribler_main.py

```diff
@@ -24,7 +24,7 @@
 import urllib
 
 original_open_https = urllib.URLopener.open_https
-import M2Crypto  # Not a useless import! See above.
+
```

Comment:
@devos50 is sure this is useless now. It was used in versioning.py which is no longer active. Also this was used for a bug in M2Crypto 0.16 and that is fixed by now.